src/core/screen.js

```diff
@@ -81,6 +81,9 @@ export class Screen extends Phaser.Scene {
         data.config && this.events.once("create", nextPage(this));
 
         if (this.scene.key !== "loader" && this.scene.key !== "boot") {
+            if (!this.scene.key.includes("shop")) {
+                this.setStatsScreen(this.scene.key);
+            }
```

Comment:
😢

Reply:
what if the scene was called "stashopen" or something

Reply:
Or DROP TABLE GAMES; 

?

Reply:
⬆️